### DISCUSSION TITLE
2023-02-22 Weekly Meeting 96 Agenda/Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#360] [General]
Weekly Meeting 96 is happening on Wednesday, February 22, 2023 from 9:00 AM PST(17:00 UTC) - 10:00 AM PST (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Discussions: 
- Retro part II is today, continuing on our work from 2/1/23, a number of action items emerged and were then refined during the OMI Champions meeting on 02/03/23
-  [100 week celebration](https://github.com/omigroup/omigroup/discussions/332#discussioncomment-4779596)? 
  - A time poll was posted, I am happy to find a time now. I can meet weekly, let's put something special together. 
- https://github.com/omigroup/omigroup/issues/358

### Working Group Updates:
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
  - shifting to 501c, applying for a grant (Epic Mega Grants), MSF-update itself 
- [UX Research](https://github.com/omigroup/ux-research) 
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - update draft protocol submission #356 
  - omi_personality
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [OMI Website update](https://github.com/omigroup/website)
- any others?

#### Next week: 

- review action items from the retro(s) 
- OMI 100 BUILD

### Future demo themes: 
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo
- `IPSME` and Me
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
M3's templates
magicks waitlist

**Reply by knev** [reply]
IPSME "Getting Started" published: [https://ipsme.dev/ipsme/getting_started.html](https://ipsme.dev/ipsme/getting_started.html)

**Comment by mrmetaverse** [comment]
## MSF 

- the great: they are really listening! a lot more inclusivity of OMI in the discussion as of late, being more mindful of how to include smaller orgs and non-profits, letting folks in under a no-dues status. 
  - funksoup, avatarjoy, evo, and others actively engaging 
- still some clarity needed around what an SSO is, and how OMI may fall under the standards-setting-organization category. 
  - Maybe OMI falls under a SSO Principal members, no dues, but still vote. 
- also a pathway to membership as an individual (no org, no OMI, just solo) but more clarity is needed. 
- If you are an individual not attached to an org, you can't really apply to be part of MSF, so they are recommending people go through OMI. This is a lift for our org, and a great opp for individuals. 
  - But what happens then in the voting process? Do their votes get dilluted? 
    - If they come in as an individual they get 1 vote theoretically 
    - OMI only gets 1 vote as an org too 
    - so what happens with the voting process then? If 50 indivudals could get 50 votes against OMI's one vote? 
      - meeting tonight for the APAC region. 

There are effects to the organization, of more work to do and added stress. We may need support. 
- We are looking into grants 
- if we flip our opencollective to 501c3 we can take in grants 
  - great for our ability to maintain our documentation, issue bounties, sustain our relationship(s) with other orgs like MSF
  - Epic Mega Grants is a relatively easy way for us to do that. An application is getting drafted. 
  - these grants are usually in the 20k range or so. 
  - What are the small stipend roles within our community that can help us grow effectively and achieve our goals?

**Reply by mrmetaverse** [reply]
We have a lot of senior level individuals who define strategies but no support for implementation. Newcomers to the Metaverse could gain experience working with OMI on our efforts. Funding would unlock our capacity to grow, and deliver, while making an impact for others who need experience.

**Reply by mrmetaverse** [reply]
1 Do we convert to 501c3? 
  - do we stay under the open collective fiscal sponsorship umbrella (short term solution)
    - we are already under 501c6 designation, migration is a lighter lift than complete conversion. 
  - convert ourselves to a freestanding 501c3 (long term solution) 
    - have to file in a State first as a non-profit 

2 Do we apply for grants? 
  - Epic Mega grant 
  - our friends at IPFS 
  - our friends at MATIC, etc.

**Reply by mrmetaverse** [reply]
- define Open collective requirements 
- draft grant application for review

**Reply by mrmetaverse** [reply]
We can do very lightweight experiments for now ($250-500 level max) as we're sitting on about $4K as an org total
After grants, those stipends could go to a more appropriate level for the work people put in
Realistically we've had people ask for bounties/stipends of a sort for bookkeeping, newsletters, media production/video editing, OMI 3D build events/builder asset needs, welcoming committee, outreach, admin/minutes keeping, tax/legal -- most of these tasks we are limping around on between our existing work right now
About a third of these are covered by volunteers right now but there's a lot we're missing, or that falls thru cracks
None of those bounty costs move us toward other goals like UX research, portal jump events etc

**Reply by mrmetaverse** [reply]
get ahold of OC and be super specific about process to ensure we understand.

**Comment by mrmetaverse** [comment]
From our github, link to all the other githubs of teams contiributing

if you build something for OMI, we want to spread the word and drive business/engagement your way

**Comment by mrmetaverse** [comment]
## glTF extensions
- opened PRs with Khronos group, tons of discussion from Khronos and Microsoft. 
- come to the glTF meeting tomorrow for a full update on all the amazing stuff that is happening!  (5:30 ET) 
- recent notes: https://github.com/omigroup/gltf-extensions/discussions/144

**Reply by mrmetaverse** [reply]
don't want to become a godot mono-culture, seeking more diverse participation (other engines)

**Reply by mrmetaverse** [reply]
the people at OMF want to move forward with USD as the standard. Interested in loading glTF's inside of USD.

**Reply by fire** [reply]
As far as I know, the USD folks want the metaverse to import and implement USD and GLTF.

Game engine tend to take the strategy they would ingest USD, convert to an internal format and export USD. This remove one of the major advantage of USD of keeping the originals work intact.

Game engines tend to take the strategy of implementing all possible imports, and this is hard for small indie game engines and engines like Godot Engine.

**Comment by mrmetaverse** [comment]
## Media 

- seeking more EU friendly meeting time

**Reply by mrmetaverse** [reply]
a person volunteered to run our newsletter 
want to run a new strategy meeting 
ideally a time next week to meet. 
mm to help drive that

**Comment by mrmetaverse** [comment]
## Portals proposed 

We discussed #358

**Reply by mrmetaverse** [reply]
created a new channel omi-portals-pro-tem

**Comment by mrmetaverse** [comment]
Dr. Kim Nevelsteen has a major announcement! Check out this video and we will host an AMA!

**Comment by mrmetaverse** [comment]
## M3 Booths 

Get your booths in! M3 was kind enough to provide templates for creating your own vendor booth for the upcoming virtual Metaverse conference they are hosting. Deadline is this weekend!
